modules/null/federation/variables.tf

```diff
@@ -20,6 +20,36 @@ variable "hub_info" {
   description = "Hub info"
 }
 
+variable "gws_proxy_info" {
```

Comment:
Maybe change gws->gw

Reply:
I want it to be equivalent to the existing "gws_info" variable. What do you think?